conda-forge.yml

```diff
@@ -1,3 +1,5 @@
+azure:
+  free_disk_space: true
```

Comment:
The CUDA 11 arch builds were failing due to space limitations. Adding this config setting seems to fix that issue